spec/system/claims/terms_page_spec.rb

```diff
@@ -0,0 +1,162 @@
+require "rails_helper"
+
+RSpec.describe "Terms Page", type: :system, service: :claims do
+  scenario "User visits the terms page" do
+    given_i_am_on_the_terms_page
+    then_i_can_see_the_terms_page
+  end
+
+  private
+
+  def given_i_am_on_the_terms_page
+    visit claims_terms_path
+  end
+
+  def then_i_can_see_the_terms_page
+    within(".govuk-heading-l") do
+      expect(page).to have_content("Terms and conditions")
+    end
+
+    expect(page).to have_content("Using our website")
+    expect(page).to have_content(
+      "Please read these Terms of Use (“General Terms”) carefully before using "\
+      "this Claim funding for mentors (the “Service”).\n"\
+      "This Service is maintained for your personal use and viewing. Access and "\
+      "use by you of this Service constitutes your acceptance of these General Terms. "\
+      "This takes effect from the date on which you first use this website. If "\
+      "you do not agree to these terms, you must not use this website.\n"\
+      "If we change these General Terms we will post the revised document here "\
+      "with an updated effective date. If we make significant changes to these "\
+      "terms, we may notify you by other means such as sending an email or "\
+      "posting a notice on our home page.",
+    )
+
+    expect(page).to have_content("Information About Us")
+    expect(page).to have_content(
+      "This Service is operated by the Department for Education (“we”, “our”, or “us”). "\
+      "We are a central government department and have our registered office at "\
+      "Sanctuary Buildings, 20 Great Smith Street, SW1P 3BT.\n"\
+      "You can contact us using the following email address: "\
+      "becomingateacher@digital.education.gov.uk",
+    )
+
+    expect(page).to have_content("Access to the Service")
+    expect(page).to have_content(
+      "We will not be liable if for any reason the Service is unavailable at any "\
+      "time or for any period. From time to time, we may restrict access to all "\
+      "or some parts of the Service to users who have registered with us.\n"\
+      "If you choose, or are provided with, a user verification code, password or "\
+      "any other piece of information as part of our security procedures, you must "\
+      "treat such information as confidential, and you must not disclose it to any third party.\n"\
+      "We reserve the right to restrict or deny you access to all or some part "\
+      "of the Service if in our opinion you have failed to comply with these Terms and Conditions.\n"\
+      "The Department for Education withholds the right to withdraw without notice "\
+      "listings that breach these terms and to withdraw the access of users who breach them.",
+    )
```

Comment:
I'm not sure how valuable these tests are. Is it really necessary to test the static content of these pages?

I feel like we could probably get just as much value from these tests by just checking that the page loads and has the expected title.